src/TestLoader.ts

```diff
@@ -119,7 +117,7 @@ class TestLoader {
     this.log.info("TestLoader disposed");
   }
 
-  private async handleCreatedFile(uri: vscode.Uri, matcher: Matcher) {
```

Comment:
A few more signature changes here in the removal of the async keyword in this file.